app/assets/javascripts/angular/services/graph_refresher.js

```diff
@@ -21,9 +21,10 @@ angular.module("Prometheus.services").factory('GraphRefresher',
         cache: false
       }).then(function(payload, status) {
         data = payload.data;
+        var errMsg;
```

Comment:
Still, for readability I think it's better the other way around. Do you know if that check can be disabled, or maybe there's good arguments for not doing so?